src/amo/pages/Feedback/index.js

```diff
@@ -0,0 +1,478 @@
+/* @flow */
+/* global window */
+import invariant from 'invariant';
+import * as React from 'react';
+import Textarea from 'react-textarea-autosize';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import config from 'config';
+
+import NotFoundPage from 'amo/pages/ErrorPages/NotFoundPage';
+import Page from 'amo/components/Page';
+import { sendAddonAbuseReport } from 'amo/reducers/abuse';
+import { getCurrentUser } from 'amo/reducers/users';
+import { VIEW_CONTEXT_HOME } from 'amo/constants';
+import { withFixedErrorHandler } from 'amo/errorHandler';
+import log from 'amo/logger';
+import translate from 'amo/i18n/translate';
+import Button from 'amo/components/Button';
+import Card from 'amo/components/Card';
+import Notice from 'amo/components/Notice';
+import type { AddonAbuseState } from 'amo/reducers/abuse';
+import type { UserType } from 'amo/reducers/users';
+import type { AppState } from 'amo/store';
+import type { ElementEvent, HTMLElementEventHandler } from 'amo/types/dom';
+import type { ErrorHandlerType } from 'amo/types/errorHandler';
+import type { I18nType } from 'amo/types/i18n';
+import type { DispatchFunc } from 'amo/types/redux';
+import type { ReactRouterMatchType } from 'amo/types/router';
+import { setViewContext } from 'amo/actions/viewContext';
+
+import './styles.scss';
+
+type Props = {||};
+
+type PropsFromState = {|
+  currentUser: UserType | null,
+  loading: boolean,
+  currentUser: UserType | null,
+  addonIdentifier: string,
+  abuseReport: AddonAbuseState | null,
+|};
+
+type DefaultProps = {|
+  _window: typeof window | Object,
+  _config: typeof config,
+|};
+
+type InternalProps = {|
+  ...PropsFromState,
+  ...DefaultProps,
+  dispatch: DispatchFunc,
+  errorHandler: ErrorHandlerType,
+  i18n: I18nType,
+  // `match` is used in `mapStateToProps()`
+  // eslint-disable-next-line react/no-unused-prop-types
+  match: {|
+    ...ReactRouterMatchType,
+    params: {| addonIdentifier: string |},
+  |},
```

Comment:
Ah yeah, no need to disable eslint, move it to where it should be defined instead (see above)